libkbfs/keybase_daemon.go

```diff
@@ -25,7 +25,10 @@ func (k keybaseDaemon) NewKeybaseService(config Config, params InitParams, ctx C
 		if err != nil {
 			return nil, err
 		}
-		return NewKeybaseDaemonRPC(config, ctx, log, params.Debug, params.CreateSimpleFSInstance), nil
+		return NewKeybaseDaemonRPC(
+			config, ctx, log, params.Debug, params.CreateSimpleFSInstance,
+			params.CreateGitHandlerInstance,
+		), nil
```

Comment:
Does this talk over the same socket as `kbfsfuse`? How is that multiplexed?

Reply:
It is receiving RPCs on the same socket, yes.  Multiplexing happens in the RPC server at the protocol level, because this is registered as a new protocol here: https://github.com/keybase/kbfs/pull/1165/files#diff-12bb5206b823967cfe8c244ffef2c727R277

Reply:
All right, so `kbfsfuse` won't create `gitHandler`, but `kbgit` will, so there's a unique protocol handler for each one. What would happen if they both implemented a given handler? Are there any other handlers that `kbgit` implements that duplicate `kbfsfuse`'s, where multiplexing ambiguity over the same socket could be a problem?

Reply:
Actually no, this `CreateRepo` RPC is served by the running `kbfsfuse` daemon.  We don't spin up a new instance of `git-remote-keybase` just to create the repo.  There's no duplication here; `git-remote-keybase` doesn't serve any RPCs.

Reply:
(Specifically, `InitParams.CreateGitHandlerInstance` is never set by `git-remote-keybase`, but it is set by `kbfsfuse` and `kbfsdokan`.)

Reply:
Cool, perfect, thanks.

Reply:
Oh, another point just for clarification: `git-remote-keybase` (or anything running in `SingleOp` mode) doesn't introduce itself to the keybase service as being a KBFS instance.  So the service knows not to route any KBFS-related RPCs over that connection.